docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -0,0 +1,115 @@
+
+Pieces Copilot+ is now temporally grounded in exactly what you have been working on, which reduces the need for manual context input, enhances continuity in developer tasks, reduces context switching, and enables more natural, intuitive interactions with the Pieces Copilot.
+
+## The Workstream Pattern Engine
+
+The Live Context in the Pieces Copilot comes from the Workstream Pattern Engine, which shadows your workflow on an operating system level. Whether you're on macOS, Windows, or Linux, your workflow will be captured, processed, and stored on-device for access at a later date. The Workstream Pattern Engine applies several on-device and real-time algorithms to this data to proactively capture the most important information as you go about your workday, wherever you are— your browser, your IDE, or your collaboration tools. Over time, the Workstream Pattern Engine learns more and more about your work and becomes increasingly helpful. It even takes context from your previous copilot chats to build on the knowledge you are gathering.
+
+## Getting Started with Live Context
+
+### Enabling/Disabling the WPE
+
+In order to use Live Context in your conversations with Pieces Copilot, you will need to enable the Workstream Pattern Engine. You can disable it at any time, but remember that the Copilot will not be able to use Live Context from when you had it disabled.
+
+To enable or disable the engine, head to the Machine Learning section of the Settings page of the Pieces Desktop App. You’ll see a button to enable the Workstream Pattern Engine— hit it, and you’re good to go!
+
+![WPE Menu](/wpe/enable-disable.gif)
+
+### Using Live Context
+
+Once you've enabled the Workstream Pattern Engine, go about your usual work for a few minutes, and then head to the Copilot Chats view in the Pieces for Developers Desktop App and select “New Chat.” In the “Set Context” section, tap the option labeled “Live Context." Like before, you can leverage any of our available models, on-device or cloud, to engage with Live Context in the Pieces Copilot, and you can use it across your toolchain to carry on conversations in your favorite IDE and browser.
+
+![WPE Pipeline](/copilot/live-context/using-live-context.png)
+
+:::info
+
+The Workstream Pattern Engine must be turned on to use Live Context. We’ve made this super easy to do from wherever you’re getting started.
+
+:::
+
+You can add additional context to further tailor the conversation if you’d like.
+
+### Permissions
+
+If you’re a Mac user, you will need to update Pieces’ permissions in order to use Live Context. Windows users may disregard this step.
```

Comment:
Maybe make this a "**Note for macOS**... and don't mention windows at all. Don't wanna raise any questions explicitly around why windows can disregard

Reply:
Updated